content: Add some civilian hails. (#5755)
endless-sky/endless-sky: data/human/news.txt

			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"

> 			"Beat it, old-timer, I've got more important things to do than chat with you."

Reviewer: I really like this, it's a nice callback.

Author: Thanks! And thanks for the review.

> 			"Beat it, old-timer, I've got more important things to do than chat with you."

Reviewer 2: ```suggestion
			"Beat it, old-timer. I've got more important things to do than chat with you."
```
Another spot where you could let this slide if it sounds better with the comma. Your choice.

			"In all of human space, there hasn't been a single nuclear meltdown in fourteen years. And that was just a tiny one, hardly even worth mentioning."
			"My company's motto is, 'We bring the power of the stars down to earth, and send earthlings to the stars.' I think we should change it to something catchier."
			"Ever used a fusion reactor in your ship? Feel free to thank one of us."

> 			"The most terrifying word in my profession is 'Oops'."

Reviewer 3: ```suggestion
			"The most terrifying word in my profession is 'Oops.'"
```

			"I'm employed as the chef on a luxury cruise ship. A Star Queen. The captain tells me that Star Queens are considered the safest way to travel in outer space."
			"A rich client of mine shot a spiny-tailed gorgebeast on a safari expedition and asked me to cook it for him. The flavor profile was quite... unique."
			"I was a dish-washer when I started, and now I'm a sous-chef. And people say there's no upward mobility in the galaxy these days!"
			"Last week my dish was completely ruined; somebody had calibrated my kitchen scale with the gravity settings of another planet."

> 			"There is only one thing worse than finding a worm in an apple you're eating; finding half a worm in that apple."

Reviewer: ```suggestion
			"There is only one thing worse than finding a worm in an apple you're eating: finding half a worm in that apple."
```

			"It's criminal how cheap the fares are to get here. For around a few thousand credits, you get to see stunning sights and shocking spectacles well worth double of what you paid."
			"I was really hoping for more aliens in space. What's the point if there's no steamy interspecies romance!?"
			"Can you tell me again which planet this is? No, I'm not lost, I just don't know where I am."
			"We did a ride on an ancient spacecraft the other day. It was called something like Soyuz or Shuttle Dragon; I don't remember the name exactly, but it was horrible; my back still hurts from the bumpy ride."
		word
			`"`

news "human protester"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	name
		word
			"Protester"
	message
		word
			`"`
		word
			"Stay with global companies! Don't depend on galactic companies from other worlds!"

> 			"Humans were not built for traveling in space. Bad things happen when travelling at 25000 km per second!"

Reviewer 2: ```suggestion
			"Humans were not built for traveling in space. Bad things happen when travelling at 25,000 kilometers per second!"
```

			`"`

news "human protester"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	name
		word
			"Protester"
	message
		word
			`"`
		word

> 			"Stay with global companies! Don't depend on galaxic companies from other worlds!"

Reviewer: ```suggestion
			"Stay with global companies! Don't depend on galactic companies from other worlds!"
```

			"It's criminal how cheap the fares are to get here. For around a few thousand credits, you get to see stunning sights and shocking spectacles well worth double of what you paid."
			"I was really hoping for more aliens in space. What's the point if there's no steamy interspecies romance!?"
			"Can you tell me again which planet this is? No, I'm not lost, I just don't know where I am."
			"We did a ride on an ancient spacecraft the other day. It was called something like Soyuz or Shuttle Dragon. I don't remember the name exactly, but it was horrible; my back still hurts from the bumpy ride."

> 			"I almost missed my transport last week; Someone was wrong on the hypernet and I was in a long heated discussion to correct it."

Reviewer: ```suggestion
			"I almost missed my transport last week; someone was wrong on the hypernet and I was in a long, heated discussion to correct it."
```

> 			"Hey! Where is platform 5? I need to board there in a few minutes!"

Reviewer: ```suggestion
			"Hey! Where's platform 5? I need to board there in a few minutes!"
```

			"Have you seen my luggage? I'm sure I put it here some minutes ago."
			"We visited an exhibition about ancient military weapons; stuff like stealth fighters, artillery and tanks. It all became obsolete so fast when we got cheap spaceflight."
		word
			`"`

news "human protester"
	location

> 		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"

Author: ```suggestion
		government "Republic" "Free Worlds" "Syndicate"
```

> 		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"

Author: Not on alien planets.

> 		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"

Author: I removed Neutral and Independent, those protests make less sense there.

			"Your credits or your life! Oh you don't have either? Never mind then."
			"Don't count on the Navy to save you if me or any of my friends attack you. Those Navy goons only hang around near the rich planets."
			"Did I attack you? Well, I attacked many people, could you be a bit more specific?"
			"The size of my crimes no longer matter; one may as well hang for a sheep as for a lamb."

> 			"Hey! We have this new Piracy As A Service offering. Want someone ambushed without incriminating yourself? Just tell me who, where and how much it's worth to you."

Reviewer 3: ```suggestion
			"Hey! We have this new Piracy As A Service offering. Want someone ambushed without incriminating yourself? Just tell me who, where, and how much it's worth to you."
```

> 			"I don't get it; If I donate a kidney, then I'm some sort of hero. But if I donate 7 kidneys, then suddenly I'm some sort of monster."

Reviewer 3: ```suggestion
			"I don't get it; if I donate a kidney, then I'm some sort of hero. But if I donate 7 kidneys, then suddenly I'm some sort of monster."
```

			"You think piracy is bad and want to stop us? You and what army?"
			"Messing with the Syndicate, Does That Sound Like Fun to You?"
			"83 percent of all statistics are just made up."
			"Ask not what your planet can do for you; ask what you can do for your planet. - based on John F. Kennedy"
			"Fair is foul, and foul is fair - Shakespeare, Macbeth"

> 			"For want of a shoe the crewmember was lost. / For want of a crewmember the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."

Reviewer 2: ```suggestion
			"For want of a shoe, the crewmember was lost. / For want of a crewmember, the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
```
OR
```suggestion
			"For want of a shoe the crewmember was lost. / For want of a crewmember the ship was lost. / For want of a ship the fleet was lost. / For want of a fleet the battle was lost. And all for the want of a shoe."
```
Commas could go either way, but I think being consistent with one way or the other is a good idea.

			"Fair is foul, and foul is fair - Shakespeare, Macbeth"
			"For want of a shoe the crewmember was lost. / For want of a crewmember the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
			"Shoot for the stars, but make sure you miss or your hull might not last."
			"Clippy the paperclip was here."
			"Kilroy was here."
			"Lassy ran by."
			"Mr Pappa was here."
			"The Easter Bunny was here."
			"The Joker was here."
			"Veronica Oxygenpocket was here."
			"Vincent van Gogh was here."
			"Beethoven and Mozart made music here."
			"Foo went to a Bar."
			"Hansel and Gretel experienced fairy tales here."
			"Jane and John Doe were here."
			"Jack and Jill went up a hill here."
			"Johnny Walker and Jack Daniels had a drink here."
			"The Pink Elephant was in a room around here."

> 			"The Three Scrooges were here."

Reviewer: ```suggestion
			"The Three Stooges were here."
```

			"An elephant with painted nails was here but not seen by anyone."
			"Elvis was here. He is alive!"
			"Nessie, the Loch Ness monster, exists!"
			"Waldo was here, but where is he now?"

> 			"The buttler did it!"

Reviewer: ```suggestion
			"The butler did it!"
```

> 			"Hello Hero, have heroicly been hammering hostiles?"

Reviewer: ```suggestion
			"Hello Hero, have you heroicly been hammering hostiles?"
```

news "syndicate pa system"
	location
		government "Syndicate"

> 		not attributes station

Reviewer 3: ```suggestion
		not attributes "station"
```

	name
		word
			"Public Announcement System"
	message
		word

> 			"Attention, baggage should not be left unattended. Any unattended baggage will be auctioned within days."
> 			"Attention, factory workers please proceed to the fast-lane immigation desk. Time spent in regular immigration lanes will not be re-imbursed."
> 			"Dear spaceship captains, please be aware that our Outfit shops are open all hours of the day. There is never a reason to delay the purchase of your next syndicate equipment."
> 			"Dear traveller, we recommend you to have dinner in restaurant The Wing, you will not be disapponted."

Reviewer 4: ```suggestion
			"Attention: baggage should not be left unattended. Any unattended baggage will be auctioned within days."
			"Attention: factory workers please proceed to the fast-lane immigration desk. Time spent in regular immigration lanes will not be reimbursed."
			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
			"Dear traveller: we recommend you to have dinner in restaurant The Wing. You will not be disappointed."
```

> 			"Dear traveler: we recommend you to have dinner in restaurant The Wing. You will not be disappointed."

Reviewer: ```suggestion
			"Dear traveler: we recommend you to have dinner in our local restaurant 'The Wing'. You will not be disappointed."
```

> 			"Dear traveler: we recommend you to have dinner in our local restaurant 'The Wing'. You will not be disappointed."

Reviewer 3: These are still spoken messages, so this phrase should have quotation marks at the top and bottom.

> news "touristic pa system"

Reviewer 4: ```suggestion
news "tourist pa system"
```

	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "deep" "rich" "paradise" "tourism"
	name
		word
			"Public Announcement System"
	message
		word
			"Attention: baggage should not be left unattended. Keep an eye on your belongings at all times."
			"Note to travelers: the planetary security level has been changed to black."
			"Be aware that parking restrictions are in effect around the spaceport. Do not leave your personal transport vehicles unattended."
			"Dear travelers: the spaceport is a strictly no-smoking zone. Thank you for your consideration."
			"Dear travelers: welcome to our planet. Please remember to synchronize your personal clocks to the local time and night/day cycle duration."
			"Please stand clear of the doors when the shuttle train is departing."

> 			"This is the final boarding call for passengers Eline, John, and Robert. Please proceed to platform 3 for immediate boarding."

Author: ```suggestion
			"This is the final boarding call for passengers Eline, John, and Robert Phillips. Please proceed to platform 3 for immediate boarding."
```

> 			"This is the final boarding call for passengers Eline, John and Robert. Please proceed to platform 3 for immediate boarding."

Reviewer 4: No last names?

Author: Good point. I didn't think of a good last name yet. Do you have any suggestions?

Reviewer: https://lastnamegenerator.com/common-last-names

Author: Thanks! I added a last name.

> 			"Attention, baggage should not be left unattended. Keep an eye on your belongings at all times."
> 			"Attention, note to travellers; the planetary security level has been changed to black."
> 			"Be aware that parking restrictions are in effect around the spaceport. Don't leave your personal transport vehicles unattended."
> 			"Dear travellers, the spaceport is a strictly no-smoking zone. Thank you for your consideration."
> 			"Dear travellers, welcome to our planet. Please remember to synchronize your personal clocks to the local time and to the local night/day cycle duration."
> 			"Please stand clear of the doors when the shuttle-train is departing."
> 			"This is the final boarding call for passengers Eline, John and Robert. Please proceed to platform 3 for immediate boarding."
> 			"Dear travellers, please take the time to adjust to normal gravity."

Reviewer 4: ```suggestion
			"Attention: baggage should not be left unattended. Keep an eye on your belongings at all times."
			"Note to travellers: the planetary security level has been changed to black."
			"Be aware that parking restrictions are in effect around the spaceport. Do not leave your personal transport vehicles unattended."
			"Dear travellers: the spaceport is a strictly no-smoking zone. Thank you for your consideration."
			"Dear travellers: welcome to our planet. Please remember to synchronize your personal clocks to the local time and night/day cycle duration."
			"Please stand clear of the doors when the shuttle train is departing."
			"This is the final boarding call for passengers Eline, John, and Robert. Please proceed to platform 3 for immediate boarding."
			"Dear travellers: please take the time to adjust to normal gravity."
```

> 			"Dear travelers: please take the time to adjust to normal gravity."

Reviewer 3: Same as above.

			"There once was a man from New Boston / To fly's a dream he'd had often / He purchased a shuttle / Brought folks in to huddle / Why do merchants call this a coffin? - Lake"
			"There once was a man from New Boston / He picked the ugliest of options / He purchased a star barge / Mission cargo's too large / This shipment's starting to smell rotten... - Lake"
			"There once was a man from New Boston / His loan officer advised caution / He purchased a sparrow / Took off like an arrow- / That sparrow soon turned up at auction. - Lake"

> 			`A captain with a troubled past / Passed through this sytem very fast / Was playing a game / Had no proper name / The captain was called "First Last".`

Reviewer 4: ```suggestion
			`A captain with a troubled past / Passed through this system very fast / Was playing a game / Had no proper name / The captain was called "First Last".`
```

> 			`A captain with a troubled past / Passed through this system very fast / Was playing a game / Had no proper name / The captain was called "First Last".`

Reviewer 3: ```suggestion
			`A captain with a troubled past / Passed through this system very fast / Was playing a game / Had no proper name / The captain was called "First Last."`
```

			"You think piracy is bad and want to stop us? You and what army?"
			"Messing with the Syndicate, Does That Sound Like Fun to You?"
			"83 percent of all statistics are just made up."
			"Ask not what your planet can do for you; ask what you can do for your planet."
			"Fair is foul, and foul is fair - Shakespeare, Macbeth"
			"For want of a shoe, the crewmember was lost. / For want of a crewmember, the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
			"Shoot for the stars, but make sure you miss or your hull might not last."

> 			"Clippy the paperclip was here."

Reviewer 3: How about BonziBuddy?

Author: I guess some things are best forgotten. :-)

			"You think piracy is bad and want to stop us? You and what army?"
			"Messing with the Syndicate, Does That Sound Like Fun to You?"
			"83 percent of all statistics are just made up."

> 			"Ask not what your planet can do for you; ask what you can do for your planet. - based on John F. Kennedy"

Reviewer 4: Real people (not to mention, former politicians)?

Author: I'll remove the reference. It is indeed unlikely that some graffiti artist would add a proper reference.

> 			"Ask not what your planet can do for you; ask what you can do for your planet. - based on John F. Kennedy"

Author: ```suggestion
			"Ask not what your planet can do for you; ask what you can do for your planet."
```

			"Fair is foul, and foul is fair - Shakespeare, Macbeth"
			"For want of a shoe, the crewmember was lost. / For want of a crewmember, the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
			"Shoot for the stars, but make sure you miss or your hull might not last."
			"Clippy the paperclip was here."
			"Kilroy was here."
			"Lassy ran by."
			"Mr Pappa was here."
			"The Easter Bunny was here."
			"The Joker was here."
			"Veronica Oxygenpocket was here."
			"Vincent van Gogh was here."
			"Beethoven and Mozart made music here."
			"Foo went to a Bar."
			"Hansel and Gretel experienced fairy tales here."
			"Jane and John Doe were here."
			"Jack and Jill went up a hill here."
			"Johnny Walker and Jack Daniels had a drink here."
			"The Pink Elephant was in a room around here."

> 			"The Three Stooges were here."

Reviewer: I'm not really sure about these kinds of lines. They seem kind of random and out of place.

Author: Specifically lines 1350 to 1358? Or also some of the other lines here?

Reviewer: A lot of the lines in this part are kind of strange to me.

Author: I removed a number of those lines. Please let me know if any of the remaining lines are still out of place.
